nucypher/blockchain/eth/sol/source/contracts/StakingEscrow.sol

```diff
@@ -177,37 +172,40 @@ contract StakingEscrow is Issuer {
     * @notice Set policy manager address
     **/
     function setPolicyManager(PolicyManagerInterface _policyManager) external onlyOwner {
-        require(address(policyManager) == address(0), "Policy manager can be set only once");
-        require(_policyManager.escrow() == address(this),
-            "This escrow must be the escrow for the new policy manager");
+        // Policy manager can be set only once
+        require(address(policyManager) == address(0));
+        // This escrow must be the escrow for the new policy manager
+        require(_policyManager.escrow() == address(this));
         policyManager = _policyManager;
     }
 
     /**
     * @notice Set adjudicator address
     **/
     function setAdjudicator(AdjudicatorInterface _adjudicator) external onlyOwner {
-        require(address(adjudicator) == address(0), "Adjudicator can be set only once");
-        require(_adjudicator.escrow() == address(this),
-            "This escrow must be the escrow for the new adjudicator");
+        // Adjudicator can be set only once
+        require(address(adjudicator) == address(0));
+        // This escrow must be the escrow for the new adjudicator
+        require(_adjudicator.escrow() == address(this));
         adjudicator = _adjudicator;
     }
 
     /**
     * @notice Set worklock address
     **/
     function setWorkLock(WorkLockInterface _workLock) external onlyOwner {
-        // Two-part require...
-        require(address(workLock) == address(0) &&  // Can't workLock once it is set.
-            _workLock.escrow() == address(this));  // This is the escrow for the new workLock.
+        // WorkLock can be set only once
+        require(address(workLock) == address(0));
+        // This escrow must be the escrow for the new worklock
+        require(_workLock.escrow() == address(this));
```

Comment:
Are two requires better than a single two-part require?

Reply:
Not really, they are the same, it's just unification with other similar functions